docs/starting-from-scratch.md

```diff
@@ -1,142 +1,449 @@
 ---
 id: starting-from-scratch
-title: Getting Started From Scratch
+title: Starting From Scratch
 sidebar_label: Starting from scratch
 ---
 
-A simple example of how to use webpack with [single-spa](https://github.com/joeldenning/simple-single-spa-webpack-example/blob/master/README.md). Check out the docs to learn more about single-spa itself.
+## Project Overview
+single-spa allows you to build micro frontends that coexist and can each be written with their own framework. This will allow you and your team to: 
 
-#### Note: You do not have to use webpack to use single-spa
-Since single-spa is capable of operating without using webpack, don't get too hung up on this. 
+1) Use multiple frameworks on the same page. [see the single-spa ecosystem for more info](https://single-spa.js.org/docs/ecosystem.html#docsNav)
+2) Write code using a new framework, without rewriting your existing application
+3) [Lazy load](https://en.wikipedia.org/wiki/Lazy_loading) code for improved initial load time
 
-Check out [this blog post]() to learn more.   
+For this tutorial, we will be focused on creating a one code repo, one build single-spa application from scratch using React and Angular1(AngularJS) with Webpack. 
 
-## Step One: choose a module loader.
-Your module loader / bundler is the library you’ll use to [lazy load](https://webpack.js.org/guides/lazy-loading/) code. I recommend either [Webpack](https://webpack.js.org/) or [JSPM](https://jspm.io/), if you’re starting from scratch.
+It is important to note that you do not need to use webpack to use single-spa. 
+<!-- Would love some guidance here -->
+It is simply an option that ...........? 
 
-If you go with Webpack, try to use Webpack 2 if you can, since it has support for promise-based [lazy loading](https://webpack.js.org/guides/lazy-loading/). This will make things easier for you later on, since single-spa requires that your loading functions return promises. If you can’t use Webpack 2, getting single-spa to lazy load your code with Webpack 1 will require some boilerplate code.
+You can find the completed [project repo here](). Read more about [seperating applications](https://single-spa.js.org/docs/separating-applications.html) using single-spa. 
 
-JSPM/SystemJS has worse documentation than Webpack, but is a great solution for module loading if you can get past that. I recommend using jspm@0.17 — it’s still in beta but has been worked on for over a year and at Canopy we find it stable enough to use in production.
+Be sure to read through the [singe-spa docs](https://single-spa.js.org/), check out the other tutorials and the [single-spa github](https://github.com/CanopyTax/single-spa) for more info.
```

Comment:
typo: `single-spa` is spelled wrong

Reply:
Should "other tutorials" be a link? Or which tutorials are you talking about?

Reply:
I am planning to link the "migrating existing applications" tutorial and eventually a tutorial using npm packages - the "medium" difficulty project. So yes, it will be a link eventually.